.github/workflows/test_pull_request_formatting.yml

```
@@ -42,3 +45,13 @@ jobs:
             }
 
             return pr.data;
+
```

```suggestion

      - name: Check out repository code
        uses: actions/checkout@v4

```

If the PR-metadata check would fail, checking out the code doesn't even need to happen...